sdk/ai/azopenai/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- `MongoDBChatExtensionParameters.Authentication`'s type has been changed to a `OnYourDataUsernameAndPasswordAuthenticationOptions`. (PR#23620)
```

Comment:
The current change is breaking, so this should move to the applicable section.

Reply:
Oof, you're exactly right. Moving...